Tools/cases_generator/generate_cases.py

```diff
@@ -549,10 +549,20 @@ def write_metadata(self, metadata_filename: str, pymetadata_filename: str) -> No
                 "=",
                 ";",
             ):
-                for name, _ in self.families.items():
-                    instr = self.instrs[name]
-                    if instr.cache_offset > 0:
-                        self.out.emit(f'[{name}] = {instr.cache_offset},')
+                family_member_names: set[str] = set()
+                for family in self.families.values():
+                    family_member_names.update(family.members)
+                for instr in self.instrs.values():
+                    if (
+                        instr.name not in family_member_names
```

Comment:
Why do we need to exclude family members from this table?

Reply:
That's tradition -- the table only contains the data for family heads and is always consulted after looking up the deoptimized opcode in `_PyOpcode_Deopt`.

Reply:
> > exclude family members from this table?
>
> That's tradition

At first glance I thought this was some reference to a hypothetical family in the midst of conflict. 🙂